bookkeeper-server/src/main/java/org/apache/bookkeeper/conf/AbstractConfiguration.java

```diff
@@ -525,6 +527,15 @@ public void setZkLedgersRootPath(String zkLedgersPath) {
         setProperty(ZK_LEDGERS_ROOT_PATH, zkLedgersPath);
     }
 
+    /**
+     * @return zk enable health path
+     * */
+    public String getEnableHealthPath() {
+        String zkLedgersRootPath = ZKMetadataDriverBase.resolveZkLedgersRootPath(this);
+        String ledgerParentPath = zkLedgersRootPath.substring(0, zkLedgersRootPath.lastIndexOf("/ledgers"));
+        return String.format("%s/%s", ledgerParentPath, "enableHealthCheck");
```

Comment:
why not put the health path into the `/ledgers`?

Reply:
like this:
![image](https://user-images.githubusercontent.com/19296967/146484044-28a9ebf0-e084-4e56-9319-6fad0a6ca7bf.png)

Reply:
add a new method :

Reply:
Here, the string is divided according to the configured metadata service URI, for example:
metadataServiceUri=zk+hie rarchical://localhost:2181 ;/ bookie_ ss_ share/ledgers
Then the intercepted string path is: / bookie_ ss_ share，Like this: 
![image](https://user-images.githubusercontent.com/19296967/146484044-28a9ebf0-e084-4e56-9319-6fad0a6ca7bf.png)
@zymap